be/src/runtime/data_stream_recvr.cc

```diff
@@ -520,6 +520,8 @@ Status DataStreamRecvr::SenderQueue::add_chunks(const PTransmitChunkParams& requ
                     _short_circuit_driver_sequences.end()) {
                     continue;
                 }
+            }
+            if (item.driver_sequence >= 0) {
                 _has_chunks_per_driver_sequence[item.driver_sequence] = true;
             }
```

Comment:
Why set driver sequence when pipeline level shuffle disabled, is there any other plans?

Reply:
Because second request maybe change _is_pipeline_level_shuffle.